garden-backend-service/tests/conftest.py

```diff
@@ -0,0 +1,209 @@
+import json
+from pathlib import Path
+import shutil
+from typing import Optional
+
+from fastapi import HTTPException, Depends
+from fastapi.security import HTTPBearer
+from httpx import AsyncClient
+import pytest
+from sqlalchemy.engine import create_engine
+from testcontainers.postgres import PostgresContainer
+from unittest.mock import MagicMock
+from uuid import UUID
+
+from src.api.dependencies.auth import (
+    AuthenticationState,
+    authenticated,
+    _get_auth_token,
+)
+
+from src.config import Settings, get_settings
+from src.main import app
+from src.models.base import Base
+
+
+@pytest.fixture
+def client(patch_globus_groups):
```

Comment:
understanding check: what's the difference between this client and the previous `from fastapi.testclient import TestClient`?